pkgs/top-level/python-packages.nix

```diff
@@ -6573,7 +6573,7 @@ in {
     cudatoolkit = pkgs.cudatoolkit_10;
     cudnn = pkgs.cudnn_cudatoolkit_10;
     nccl = pkgs.nccl_cudatoolkit_10;
-    openssl = pkgs.openssl_1_0_2;
+    openssl = pkgs.openssl_1_1;
```

Comment:
You can remove this line entirely because `openssl_1_1` is the default openssl.

Reply:
What’s the policy since this is a cherry pick of a commit in master?

Reply:
This can be ignored for now. It's not worth making that change on stable.